src/lectern-client/schema-manager.ts

```diff
@@ -18,6 +19,16 @@ class SchemaManager {
   getCurrent = (): SchemasDictionary => {
     return this.currentSchema;
   };
+
+  getSubSchemasList = (): string[] => {
+    const subSchemasList: string[] = [];
+    this.currentSchema.schemas.forEach(subSchema => {
```

Comment:
avoid creating and then mutating variables when you can. If `currentSchema.schemas` is already an array, then you can probably do something like:
```typescript
return this.currentSchema.schemas.filter(s => s.name !== FileType.REGISTRATION)
```